### ISSUE TITLE
unresolved haxe.Constraints types

### ISSUE THREAD

**Issue by eliasku** [TiVo/intellij-haxe#261]
Unable to import and use `haxe.Constraints` types.

repro:
```
import haxe.Constraints.Function;

class SignalBase<T:Function> {
    public function new() {}
}
```

screenshot:
![image](https://cloud.githubusercontent.com/assets/3038174/7582681/d2dc736a-f898-11e4-875e-23af8639ce7e.png)

**Comment by romanmikhailov** [comment]
@eliasku 

```
import haxe.Function;

class SignalBase<T:Function> {
    public function new() {}
}
```

works fine.

**Comment by EBatTiVo** [comment]
Marking as enhancement because constraints haven't been implemented yet.

**Comment by eliasku** [comment]
Resolving has been fixed in 0.9.9

![image](https://cloud.githubusercontent.com/assets/3038174/13173899/40574dc8-d711-11e5-90be-d1f5583f6bdc.png)
